contact.md

```diff
@@ -19,27 +20,27 @@ Version: 2.0.1.
-| Vipps for [Shopify](https://www.vipps.no/produkter-og-tjenester/bedrift/ta-betalt-paa-nett/ta-betalt-paa-nett/shopify/) | For the plugin, not Shopify itself: Please [submit an issue on GitHub](https://github.com/vippsas/vipps-shopify) |
-| [Settlements](https://github.com/vippsas/vipps-developers/tree/master/settlements), invoices or similar | Please see [Settlements](https://github.com/vippsas/vipps-developers/tree/master/settlements) |
+| Topic                                                                                                                           | How to get help                                                                                                                                                            |
+| ------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| General questions, including changes to the customer relationship, creating eCom sale units, change of admins, etc              | Please see the [contact info for companies on vipps.no](https://www.vipps.no/kontakt-oss/bedrift/)                                                                         |
+| Applications for Vipps products                                                                                                 | Please see the [contact info for companies on vipps.no](https://www.vipps.no/kontakt-oss/bedrift/)                                                                         |
+| Suggestions for improvements to [Vipps products](https://www.vipps.no/produkter-og-tjenester/bedrift/)                          | Please use [this form](https://www.vipps.no/kontakt-oss/bedrift/)                                                                                                          |
+| Using Vipps through a partner                                                                                                   | Please [contact your partner directly](https://www.vipps.no/produkter-og-tjenester/bedrift/ta-betalt-paa-nett/ta-betalt-paa-nett/#kom-i-gang-med-vipps-pa-nett-category-3) |
+| Ordering Vipps through a partner                                                                                                | Please contact partnerbestilling@vipps.no                                                                                                                                  |
+| If you are a partner and need help                                                                                              | Please contact partnerbestilling@vipps.no                                                                                                                                  |
```

Comment:
So much white space man
https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown
look at the tables section

Reply:
Hmm.. Seems like my VSCode formatter is having a feast. I'll look into it

Reply:
fixed in bab9c39

Reply:
You seem to have fixed it in readme.md, but not here?

Reply:
🤦‍♂️Fixed in 61d029a